data.sql

```diff
@@ -0,0 +1,6 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', '2020-02-03, 10.23', true, 0);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Gabumon', '2018-11-15, 8', true, 2);
```

Comment:
- [ ] First two commands are giving a syntax error and the syntax error is that if you see in the VALUES of these first two commands then you will find that you are writing date_of_birth and weight_kg together in one single quote like this **'2020-02-03, 10.23'**, these two values should be separated by two single quotes like this **'2020-02-03', '10.23'** :+1: Kindly make this change to follow the requirements :blush:

Reply:
Ohh okay my mistake😅, thank you for your feedback!🙏